src/monster.cpp

```diff
@@ -932,7 +926,7 @@ void Monster::onThinkDefense(uint32_t interval)
 		}
 	}
 
-	if (!isSummon() && summons.size() < mType->maxSummons) {
+	if (!isSummon() && summons.size() < mType->maxSummons && hasFollowPath) {
```

Comment:
Maybe, i thougt hasFollowPath check if the creature is reachable. But it does not. I have no idéa how it should behave, either summon when it has a target or summon if target is reachable.